backend/de.metas.manufacturing/src/main/java/de/metas/manufacturing/acct/Doc_PPCostCollector.java

```diff
@@ -197,7 +192,14 @@ private Fact createFactLines(
 		dr.setM_Locator_ID(docLine.getM_Locator_ID());
 
 		final FactLine cr = fact.createLine(docLine, credit, cost.getCurrencyId(), null, cost.getValue());
-		cr.setQty(qty);
+		if (CostCollectorType.ComponentIssue.equals(getCostCollectorType()))
```

Comment:
why not sending it negated to this method when needed?

NOTE: the Qty of DR and CR shall be of opposite signs